src/video_core/vulkan_common/vulkan_device.cpp

```diff
@@ -587,6 +587,13 @@ Device::Device(VkInstance instance_, vk::PhysicalDevice physical_, VkSurfaceKHR
             ext_extended_dynamic_state = false;
         }
     }
+    if (ext_sampler_filter_minmax) {
+        if (driver_id == VK_DRIVER_ID_AMD_PROPRIETARY ||
+            driver_id == VK_DRIVER_ID_AMD_OPEN_SOURCE || driver_id == VK_DRIVER_ID_MESA_RADV) {
+            // Disable ext_sampler_filter_minmax in GCN as it is broken.
+            ext_sampler_filter_minmax = is_float16_supported;
```

Comment:
```suggestion
    if (!is_float16_supported) {
        // LOG_WARNING(..., "Blaclisting AMD GCN for ...");
        ext_sampler_filter_minmax = false;
    }
```
Let's us be more explicit that HW not supporting FP16 is the target of being blacklisted for this extension by, and allows us to log it.